www/front_src/src/Header/userMenu/index.cypress.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { mount } from '../../../../../cypress/support';
+
+import UserMenu from './index';
+
+describe('User Menu', () => {
+  beforeEach(() => {
+    cy.fixture('userMenu').as('user');
+    cy.server();
+    cy.route('GET', '**/internal.php?**', '@user').as('getUser');
+
+    cy.intercept('PATCH', 'parameters', {
+      theme: 'dark',
+    }).as('updateTheme');
+
+    mount(
+      <Router>
+        <div style={{ background: 'black', width: '100%' }}>
+          <UserMenu />
+        </div>
+      </Router>,
+    );
+  });
+  it.only('matches the current snapshot "initial menu"', () => {
+    cy.get('[data-testid=AccountCircleIcon]')
+      .as('userIcon')
+      .should('be.visible');
+
+    cy.get('@userIcon').click();
+
+    cy.get('[data-cy=popper]').as('popper').should('be.visible');
+    cy.get('@popper').contains('admin');
+    cy.get('@popper').contains('Dark');
+    cy.get('@popper').contains('Light');
+    cy.get('@popper').contains('Logout');
+
+    // cy.matchImageSnapshot();
+  });
+
+  it('switch theme mode', () => {
```

Comment:
Careful with the test name as it doesn't make much sense. It should be coherent with what we already have